lib/common/asar.js

```diff
@@ -11,19 +11,15 @@
   const cachedArchives = {}
 
   const envNoAsar = process.env.ELECTRON_NO_ASAR && process.type !== 'browser' && process.type !== 'renderer'
-  const isAsarDisabled = function () {
-    return process.noAsar || envNoAsar
-  }
+  const isAsarDisabled = () => process.noAsar || envNoAsar
```

Comment:
In future, please separate refactors and fixes into separate PRs -- or more generally, please have one topic per PR. This should have been one (comparatively small) PR for the inheritance fix and then a followup for ES6ification.

 * better commit log paper trail
 * easier to cherry-pick changes
 * makes code review easier
 * makes tracking down regressions easier

Semantic Commits follows this pattern by having both `fix` and `refactor` tags